crypto/src/main/kotlin/net/corda/v5/crypto/CompositeKeyNodeAndWeight.kt

```diff
@@ -0,0 +1,10 @@
+package net.corda.v5.crypto
+
+import java.security.PublicKey
+
+data class CompositeKeyNodeAndWeight(val node: PublicKey, val weight: Int = 1) {
```

Comment:
Can you add the overload constructor for the default weight

Reply:
done